docker-compose.dev.yaml

```diff
@@ -53,7 +53,7 @@ services:
       - POSTGRES_USER=${POSTGRES_USER}
       - POSTGRES_PASSWORD=${POSTGRES_PASSWORD}
     ports:
-      - "${POSTGRES_PORT:-5432}:5432"
+      - "${POSTGRES_PORT}:${POSTGRES_INTERNAL_PORT}"
```

Comment:
I think this is very dangereous, the internal postgres port is 5432. I will remove it

Reply:
I'm not sure how this can be dangerous, please expound. This compose file is intended to only be used for CI runs, and I don't see how mapping a port to another value is dangerous.